gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -65,6 +65,7 @@ public void apply(Project project) {
                                 .configure(ErrorProneOptions.class, errorProneOptions -> {
                                     errorProneOptions.check("Slf4jLogsafeArgs", CheckSeverity.OFF);
                                     errorProneOptions.check("PreferSafeLoggableExceptions", CheckSeverity.OFF);
+                                    errorProneOptions.check("Slf4jConstantLogMessage", CheckSeverity.OFF);
```

Comment:
Arguably this is actually still valuable for perf, maybe it should just be a warning?

Reply:
This one shouldn’t be disabled